### DISCUSSION TITLE
Why Github does not reply to my ticket ?

### DISCUSSION THREAD

**Discussion by mathieuj2024** [community/community#120995] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello, I am waiting since 2 weeks for an answer from the Github support to my ticket #2723475 but they didn't reply. I had to create a new account just for write this message because everything is blocked on my other account, I got this message: You are marked as spam, and therefore cannot authorize a third party application. What I should do to resolve the situation ?

**Comment by queenofcorgis** [comment, marked as answer]
Hi @mathieuj2024, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

Please note: The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
